src/jobs/pages/sommerjobb-designer-2023.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Sommerjobb designer 2023
+h1_title: Sommerjobb for designstudenter i 2023
+slug: sommerjobb-designer-2023
+contact_emails: mk@variant.no
+meta_title: Vi søker designstudenter til sommerjobb hos Variant!
+meta_description: Vi ser etter læreglade designere til å ha sommerjobb hos oss i 2023.
+meta_image: https://www.variant.no/images/sommerjobb_designer_2023_meta.png
+---
+
+### Hva går sommerjobben ut på?
+
+En sommerjobb i Variant er en fin mulighet til å anvende det du har lært på skolen i praksis. Det forventes ikke at du er utlært, men at du ønsker å lære mer. Det viktigste er at du bryr deg. Bryr deg om brukerne du designer for, bryr deg om kunden du leverer til og bryr deg om dine medstudenter og kollegaer.
+
+<div class="right">
+
+![Sommeransatte inne i Varianthuset](/work_images/sommerjobbere_innenfor_variant_2022.png)
+
+</div>
+
+I 2023 tilbyr vi sommerjobb i hele tre(!) byer: I Trondheim søker vi 3 designere. I Oslo 2, og i vårt nyeste kontor i Bergen ser vi etter 1.
+Vi har tro på at både kunder og sluttbrukere får det beste resultatet når vi jobber i tverrfaglige team. Derfor rekrutterer vi også utviklere parallelt. Sammen vil dere jobbe på et av våre spennende oppdrag for kundene våre. Selvfølgelig med god støtte, oppfølging og tilrettelegging fra erfarne konsulenter som ønsker at du lykkes.
+
+Årets sommervarianter fikk blant annet bryne seg på tjenestekartlegging for Opplev Oppdal, bookingtjeneste for Inatur, oppfølging av fiskehelse for Piscada og digital støtte av trafikksikkerhetsinspeksjoner i Statens vegvesen. 
+
+
+### Hvorfor jobbe i Variant?
+
+Variant er en variant av et konsulentselskap som er [raust, åpent og læreglad](https://handbook.variant.no/handbook#form%C3%A5l-og-verdier). Disse verdiene ligger til grunn for hvordan vi møter hverandre og våre kunder. Vi er en gjeng hyggelige og dyktige [mennesker](https://www.variant.no/ansatte) som ønsker å både tilegne og dele kunnskap. Sammen skaper vi løsninger som tjener samfunnet.
+
+Sommerjobben varer i fire + to uker med tre uker ferie i mellom, og vil gi god innsikt i hva det vil si å jobbe i konsulentbransjen og hvordan hverdagen i Variant er. Du kommer til å lære masse nyttig, samtidig som du har det gøy sammen med oss andre varianter. Vi er opptatt av at sommerjobben skal ha en god dose av både sosialt og faglig påfyll.
+
+<div class="left">
+
+![Sommeransatte i diskusjon](/work_images/sommerjobbere_i_mote_2022.png)
+
+</div>
+
+I Variant har alle innsyn i alt - selv som sommerstudent. Derfor trenger du selvsagt ikke å lure på hvordan [kontrakten din](https://avtaler.variant.no/avtaler/ansettelse-sommerjobb.html) vil se ut for sommeren. Lønna trenger du heller ikke å lure på. Du får 271.83 kr (eller 100e) for hver time, slik at du havner rett under fribeløpet! Dersom du har lyst til å lese mer om hva Variant står for kan du sjekke ut vår egen [håndbok](https://handbook.variant.no/).
```

Comment:
100e 😂

Reply:
All kred til @oddsve for den 😂

Reply:
Men, er du enig i at dette nå er klart for publisering @sarahse?

Reply:
Ja! Ser veldig bra ut 🤩